src/guide/transitions-list.md

```diff
@@ -0,0 +1,439 @@
+# List Transitions
+
+So far, we've managed transitions for:
+
+- Individual nodes
+- Multiple nodes where only 1 is rendered at a time
+
+So what about for when we have a whole list of items we want to render simultaneously, for example with `v-for`? In this case, we'll use the `<transition-group>` component. Before we dive into an example though, there are a few things that are important to know about this component:
+
+- Unlike `<transition>`, it renders an actual element: a `<span>` by default. You can change the element that's rendered with the `tag` attribute.
+- [Transition modes](/guide/transitions-enterleave#Transition-Modes) are not available, because we are no longer alternating between mutually exclusive elements.
```

Comment:
```suggestion
- [Transition modes](/guide/transitions-enterleave#transition-modes) are not available, because we are no longer alternating between mutually exclusive elements.
```